tests/test_ranges.py

```diff
@@ -898,3 +895,9 @@ def test_doesnt_extend_union(self):
             )
             union = range | other
             assert union == range
+
+    class TestIn:
```

Comment:
🧀 Here we already had a grouping for the class of functions under test, with further classes to encapsulate each function; so here I'm following that pattern for a this test, which was previously at the top-level.